presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -523,6 +524,11 @@ public SystemSessionProperties(
                         "Experimental: Use late materialization (including WorkProcessor pipelines)",
                         featuresConfig.isLateMaterializationEnabled(),
                         false),
+                booleanProperty(
```

Comment:
I don't think this is necessary. This change does not affect behavior or break compatibility, so there's no need to make it configurable. It also won't be "active" unless a function is already deprecated.